tools/ci/before_install.sh

```diff
@@ -11,6 +11,9 @@ else
   cp config/database.pg.yml config/database.yml
   psql -c "CREATE USER root SUPERUSER PASSWORD 'smartvm';" -U postgres
   export BUNDLE_WITHOUT=development
+
+  which bower || npm install -g bower
+  bower install --allow-root -F --config.analytics=false
```

Comment:
yes, before_install would be run for every test suite...if you can wrap this in a conditional like `if [ "$TEST_SUITE" -eq "vmdb" ]; then`, then it would avoid about 30 seconds on all the other suites.